### PR TITLE
[ContextualSaveBar] Revert some new dl changes and add context control

### PR DESCRIPTION
**Pull request by kyledurand**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/polaris-ux/issues/476

contextbar

image

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}

```

</details>

### 🎩 checklist

* [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
* [ ] Updated the component's `README.md` with documentation changes
* [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by kyledurand** [comment]
Made an update to tokens here to fix default buttons in dark mode:

Before: 
image
After:
image

**Review by alex-page — APPROVED** [review]

File: src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss [hunk 1]
```diff
@@ -87,21 +93,12 @@ $shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
...
-    @include page-content-when-not-fully-condensed {
-      padding: 0 spacing();
-    }
-
-    @include page-content-when-not-partially-condensed {
-      padding: 0 spacing();
-    }
+  .ContextControl {
+    opacity: 0.3;
+    // pointer-events: none;
```
Comment (dfmcphee) [inline src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss:101]: 🔥

File: src/utilities/frame/types.ts [hunk 2]
```diff
@@ -32,6 +32,8 @@ export interface ContextualSaveBarProps {
   discardAction?: ContextualSaveBarCombinedActionProps;
   /** Remove the normal max-width on the contextual save bar */
   fullWidth?: boolean;
+  /** Accepts a component that is ideally used to help users switch between different contexts */
```
Comment (dfmcphee) [inline src/utilities/frame/types.ts:35]: I would change this to be `Accepts a component that is used to help users switch between different contexts` to be a bit more opinionated about it.

**Review by dfmcphee — APPROVED** [review]
Two tiny comments but this looks great. 👍

File: src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss (hunk 1 above)
Comment (kyledurand) [reply src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss:101]: Whoops, we need this but I had it commented out to check paddings. Need to add it back

**Pull request merged**